tools/abel/validators/README.md

```diff
@@ -0,0 +1,192 @@
+
+Path to a data point's units in a telemetry payload. Used as the `key` for `units` in an entity type [translation](../../../onotology/docs/building_config.md#defining-translations). e.g. `pointset.points.zone_air_cooling_temperature_setpoint.unit`. Leave blank or put `no-units` if a field is a [multi-state value](../../../ontology/docs/ontology.md#multi-state-values)
+
+`DBO Standard Unit Value` *string*
+
+[Standardized dimensional unit](../../../ontology/docs/ontology.md#dimensional-units) for a standard field as defined in [Digital Buildings Ontology](../../../ontology/yaml/resources/fields). e.g. `degrees_celsius`
+
+`Raw Unit Value` *string*
+
+Unit value as reported by a [reporting
+device](../../../onotology/docs/building_config.md#reporting-physical-devices). Used for a [dimensional unit's](../../../ontology/docs/ontology.md#dimensional-units) value mapping in an entity [translation](../../../ontology/docs/building_config.md#defining-translations) e.g. `degC`
+
+### States
+
+The `States` tab defines values for [multi-state value](../../../ontology/docs/building_config.md#multi-state-values) fields defined in the `Entity
+Fields` tab.
+
+`Entity Code` *string* **required**
+
+If a field is linked to a [virtual entity](../../../ontology/docs/building_config.md#virtual-devices) and its telemetry data is reported
+by some [reporting enity](../../../ontology/docs/building_config.md#reporting-physical-devices), then the `Entity Code` column must be the virtual entity code and the `reporting entity code` is the code of the
+entity reporting the field's data. This may be a gateway or some other IoT connected entity.
+
+`Entity Guid` *string*
+
+[UUID4](https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random)) [identifier](../../../ontology/docs/building_config.md#identifiers) for an entity corresponding to the entity in the Entity Code column.
+
+`Reporting Entity Field` *string* **required**
+
+Name of the [entity field](#entity-fields) which [reports](../../../ontology/docs/building_config.md#reporting-physical-devices) a [multi-state value](../../../ontology/docs/ontology.md#multi-state-values) in its telemetry payload.
+
+`DBO Standard State` *string* **required**
+
+Stanardized [multi-state value](../../../ontology/docs/ontology.md#multi-state-values) name as defined in [Digital Buildings Ontology](https://github.com/google/digitalbuildings/blob/master/ontology/yaml/resources/states/states.yaml). e.g. `ON`
+
+`Payload State` *string* **required**
+
+Raw [multi-state value](../../../../../../ontology/docs/ontology.md#multi-state0-values) for a data point in the telemetry payload for a device. e.g. `on`
+
+### Connections
+
+Connections define the [device
+relationships](../../../ontology/docs/building_config.md#device-relationships) that exist between two entities
+in a building. Connections to a building such as a building having a `CONTAINS`
+relationship to a floor do not need to be defined as they are implied.
```

Comment:
this is not enough, I think you need to mention something like `For Google buildings, this information is already in the system, there is no need to re-populate it`

Reply:
I added an extra sentence.